File: 351004/Beliaev/restapp/publisher/src/main/java/com/example/publisher/dto/request/NoteRequestTo.java
package com.example.publisher.dto.request;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class NoteRequestTo {
    @NotNull
    private Long articleId;

    @Size(min = 2, max = 2048)
    private String content;
}

File: 351002/Abrashin/publisher/src/main/kotlin/com/example/distcomp/dto/request/NoteRequestTo.kt
package com.example.distcomp.dto.request

import com.fasterxml.jackson.annotation.JsonRootName
import jakarta.validation.constraints.Size

@JsonRootName("note")
data class NoteRequestTo(
    var tweetId: Long? = null,
    var country: String? = null,
    @field:Size(min = 2, max = 2048)
    var content: String? = null
)
